### PR TITLE
Fix some weird zeppelin tests.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: test/externalTests.sh [hunk 1]
```diff
@@ -41,6 +41,11 @@ DIR=$(mktemp -d)
     echo "Running Zeppelin tests..."
     git clone --depth 1 https://github.com/OpenZeppelin/zeppelin-solidity.git "$DIR"
     cd "$DIR"
+    # Fix some things that look like bugs
+    sed -i -e 's/let token = await ERC827TokenMock.new();/let token = await ERC827TokenMock.new(accounts[0], 0)/;' test/token/ERC827/ERC827Token.js
+    sed -i -e 's/CappedCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0)/CappedCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0, this.token.address)/' test/crowdsale/CappedCrowdsale.test.js
+    sed -i -e 's/RefundableCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0, { from: owner })/RefundableCrowdsale.new(this.startTime, this.endTime, rate, wallet, 0, this.token.address, { from: owner })/' test/crowdsale/RefundableCrowdsale.test.js
```
Comment (axic) [inline test/externalTests.sh]: Checked in the source, these two look reasonable. Strange though it doesn't fail on node 8?!

File: test/externalTests.sh [hunk 2]
```diff
@@ -41,6 +41,11 @@ DIR=$(mktemp -d)
     echo "Running Zeppelin tests..."
     git clone --depth 1 https://github.com/OpenZeppelin/zeppelin-solidity.git "$DIR"
     cd "$DIR"
+    # Fix some things that look like bugs
+    sed -i -e 's/let token = await ERC827TokenMock.new();/let token = await ERC827TokenMock.new(accounts[0], 0)/;' test/token/ERC827/ERC827Token.js
```
Comment (axic) [inline test/externalTests.sh]: I think this line should actually be removed in the source?

File: test/externalTests.sh (hunk 1 above)
Comment (axic) [reply test/externalTests.sh]: Does web3.js behave differently depending on node version?

Comment (chriseth) [reply test/externalTests.sh]: I have NO idea...

File: test/externalTests.sh (hunk 2 above)
Comment (chriseth) [reply test/externalTests.sh]: why?

Comment (chriseth) [reply test/externalTests.sh]: if you remove the line, you essentially disable the test. if you modify it seems to fix the test...

Comment (axic) [reply test/externalTests.sh]: `beforeEach` creates the `token`. This given test sets the `allowance` for `accounts[1]` on said token.

Comment (chriseth) [reply test/externalTests.sh]: ok, sounds good!

**Comment by chriseth** [comment]
Done.

File: test/externalTests.sh (hunk 2 above)
Comment (axic) [reply test/externalTests.sh]: I did test it locally before suggesting it.

**Review by axic — APPROVED** [review]

**Comment by axic** [comment]
@maraoz @frangio @frozeman any idea why web3.js or zeppelin would ignore missing constructor parameters on Node 8 and succeed, while it would fail on Node 6?

**Comment by frozeman** [comment]
Which version is it using ? 0.20.x?
It’s certainly a weird behavior, what exact error are you getting in node 6?

**Comment by axic** [comment]
Errors start here: https://travis-ci.org/ethereum/solidity/jobs/340414933#L3458

**Comment by frozeman** [comment]
The version is even 0.16 I would suggest updating to at least 0.20
I think I had that error once but don’t remember why anymore. Can you test updating and we see if it fixes it

**Comment by axic** [comment]
The strange thing is with Node 8 it doesn't complain (running it at upstream), but those methods indeed seem to be lacking those fields. We've fixed here with manually replacing those :)

Will report upstream and bug them to upgrade web3.js.

**Comment by frozeman** [comment]
Ideally to 1.0.0-beta... it’s more reliable for that stuff

**Pull request merged**